### DISCUSSION TITLE
Plex: no option to manage/add library after install.

### DISCUSSION THREAD

**Discussion by Outsmart2383** [community-scripts/ProxmoxVE#931] [Q&A] [answered]
After running the Plex installation script, when accessing the Web UI via `http://CONTAINERIP:32400/web` I get error: `Not authorised. You do not have access to this server.`

If accessing via  `http://CONTAINERIP:32400` (i.e. with `/web` ommitted) I get taken to a Plex Web UI interface, but there are no options for library management.

Initially I had immediately set the container up behind NPM, and then read that this could cause issues with libary management, but as above, when accessing directly via IP the options for library management still aren't there.

I've tried removing all authorised devices from my Plex account in case that was causing issues, but this has had no effect. I've also tried reinstalling a few times in case there had been an issue arising during deployment.

On deployment I selected to apply advanced options, as I like to configure root passwords, and apply a static IP address and VLAN to my containers, but I was sure to select the correct distribution (`Ubuntu 22.04`).

**Comment by Outsmart2383** [comment, marked as answer]
Ahhh. so it was in fact the configuration of the VLAN that was causing the issue. I was trying to access the web UI from my PC which is on a different VLAN to my homelab services.

**Comment by nicedevil007** [comment]
what was changed by yourself to get it up and running between multiple VLANs?

**Reply by Outsmart2383** [reply]
I've slept since then, but if memory serves me correctly, I chucked it on the same VLAN as my PC for the initial set up/authentication, and then was able to move it back to the other VLAN.

**Reply by nicedevil007** [reply]
> I've slept since then, but if memory serves me correctly, I chucked it on the same VLAN as my PC for the initial set up/authentication, and then was able to move it back to the other VLAN.

Ahhahaha, that was what I was planning to do now. I already got it working on the same subnet, there wasn't any blocking part on my firewall so it must be plex rated stuff and I guess it is because you can tell plex which networks should be recognized as "local"
